Gemfile.lock

```diff
@@ -58,13 +50,14 @@ GEM
       rack (>= 0.9.0)
     binding_of_caller (0.7.2)
       debug_inspector (>= 0.0.1)
-    bootstrap-sass (2.3.2.0)
-      sass (~> 3.2)
+    bootstrap-sass (3.3.7)
+      autoprefixer-rails (>= 5.2.1)
```

Comment:
It looks like this is being required here by bootstrap-sass. Do we need to require it above ? Are we using it separately ?